elements/rh-site-status/rh-site-status.css

```diff
@@ -0,0 +1,21 @@
+:host {
+  display: block;
+}
+
+a {
```

Comment:
Looks like this component is currently missing focus and hover styles. 

The [DP docs](https://deploy-preview-1507--red-hat-design-system.netlify.app/elements/site-status/style/#interaction-states) show interaction states (hover = underline, focus = underline + outline).